src/chrome/content/rules/Nexway.com.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="Nexaway.com">
```

Comment:
Would you like to add more targets or keep it `(partial)`?

Reply:
I would prefer to keep it `partial` since my intention is to fix the typo here.